packages/font-glyphs/src/letter/latin/lower-g.ptl

```diff
@@ -217,6 +217,7 @@ glyph-block Letter-Latin-Lower-G : begin
 
 	select-variant 'cyrl/de.BGR' (shapeFrom -- 'g') (follow -- [conditional-follow SLAB 'g/single/autoSerifed/slab' 'g/single/autoSerifed/sans'])
 	alias 'cyrl/de.SRB' null 'cyrl/de.BGR'
+	alias 'armn/co' 0x581 'cyrl/de.BGR'
```

Comment:
I suggest putting this into `letter/armenian` but create a new file (say, `letter/armenian/aliases.ptl`)